Assets/UnityIO.cs

```diff
@@ -0,0 +1,48 @@
+using UnityEngine;
+using System;
+using System.IO;
+using System.Collections;
+
+public class UnityIO
+{
+	public class Response {
+		public string error;
```

Comment:
Can you explain with comments how `error` should be interpreted? If a request is successful, is `error` set to `null`, `""`, `"Success"`, etc.